src/data/salesPortal/products/negativeTestDataAPI.ts

```diff
@@ -0,0 +1,129 @@
+import { MANUFACTURERS } from "./manufacturers";
+import { IProductTestCase } from "./positiveTestDataAPI";
+
+export const negativeTestTitles = {
+  NAME_EMPTY: "Попытка создать продукт с пустым именем",
+  NAME_TOO_SHORT: "Имя продукта короче 3 символов",
+  NAME_TOO_LONG: "Имя продукта длиннее 40 символов",
+  NAME_INVALID_CHARS: "Имя продукта содержит недопустимые символы",
+  MANUFACTURER_EMPTY: "Попытка создать продукт без производителя",
+  PRICE_ZERO: "Цена продукта равна 0",
+  PRICE_TOO_HIGH: "Цена продукта больше 99999",
+  AMOUNT_NEGATIVE: "Количество меньше 0",
+  AMOUNT_TOO_HIGH: "Количество больше 999",
+  NOTES_TOO_LONG: "Notes превышает 250 символов",
+  NOTES_INVALID_CHARS: "Notes содержит запрещённые символы < или >",
+};
+
+export const negativeTestData: IProductTestCase[] = [
+  {
+    title: negativeTestTitles.NAME_EMPTY,
+    data: {
+      name: "",
+      manufacturer: MANUFACTURERS.SONY,
+      price: 100,
+      amount: 10,
+      notes: "Valid notes",
+    },
```

Comment:
Можно немного упростить
```suggestion
    title: negativeTestTitles.NAME_EMPTY,
    data: generateProductData({  name: "" })
```